kloppy/tests/test_epts.py

```diff
@@ -41,6 +42,13 @@ def test_regex(self):
 
         assert result is not None
 
+    def test_provider_name_recognition(self):
+        base_dir = os.path.dirname(__file__)
+        with open(f"{base_dir}/files/epts_metrica.xml", "rb") as metadata_fp:
+            metadata = load_metadata(metadata_fp)
+
+        assert metadata.provider is Provider.METRICA
```

Comment:
This should be ‘==‘ instead of ‘is’. ‘is’ checks if variables point to the same object while ‘==‘ checks if they have the same value. In this case we need check values.